.github/workflows/pr-demo-deploy.yml

```diff
@@ -57,8 +57,7 @@ jobs:
           echo Creating dokku app ${{ github.head_ref }}
           ssh dokku@$DOKKU_HOST -C "dokku apps:create ${{ env.SUBDOMAIN }}" | true
           echo The url will be ${{ github.head_ref }}.pcpink.co.uk
-          ssh dokku@$DOKKU_HOST -C "dokku git:initialize ${{ env.SUBDOMAIN }}"
-          ssh dokku@$DOKKU_HOST -C "dokku git:set ${{ env.SUBDOMAIN }} deploy-branch ${{ github.head_ref }}"
```

Comment:
The deploy branch needs to stay, that's very important because that's the branch which gets tested.
Am I missing something?